lib/main.js

```diff
@@ -189,6 +200,21 @@ export default {
     });
   },
 
+  async getProgram(Linter, configurationPath) {
+    let program;
+    const tsconfigPath = path.resolve(path.dirname(configurationPath), 'tsconfig.json');
+    try {
+      const stats = await statAsync(tsconfigPath);
+      const isFile = stats.isFile();
+      if (isFile) {
+        program = Linter.createProgram('tsconfig.json', path.dirname(configurationPath));
+      }
+    } catch (err) {
+      // no-op
+    }
+    return Promise.resolve(program);
```

Comment:
Fixed in https://github.com/AtomLinter/linter-tslint/pull/157/commits/9bed459f8572a3213893836b2a8e55dd700a0d1b